src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -575,7 +574,7 @@ async function signOut() {
 ```
 
 You can sign out users from all devices by adding global sign-out. It also invalidates all refresh tokens issued to an user. The user's current access and Id tokens remain valid until they expire (access and Id tokens expire one hour after they are issued).
-
+ 
```

Comment:
Not sure on this last sentence above - is this telling me that even if I add global sign-out the user's access will still be possible for 1 hour after they sign out?

Reply:
on the "other" devices yes - until their "Refres" token expires. Which is something they configure in their backend.